addons/account/models/account_move.py

```diff
@@ -1065,6 +1064,7 @@ def _compute_bank_partner_id(self):
                 move.bank_partner_id = move.commercial_partner_id
             else:
                 move.bank_partner_id = move.company_id.partner_id
+            move.invoice_partner_bank_id = move.bank_partner_id.bank_ids and move.bank_partner_id.bank_ids[0]
```

Comment:
To avoid multicompany issue you could should add a check on the bank company_id 
`move.bank_partner_id.filtered(lambda bank: bank.company_id.id in (False, move.company_id.id))`

Reply:
@agr-odoo I updated the commit, thank you :pray: